src/Illuminate/Database/DatabaseManager.php

```diff
@@ -396,6 +396,19 @@ public function extend($name, callable $resolver)
         $this->extensions[$name] = $resolver;
     }
 
+    /**
+     * unset an extension connection resolver.
+     *
+     * @param  string  $name
+     * @return void
+     */
+    public function forgetExtend($name)
+    {
+        if (isset($this->extensions[$name])) {
+            unset($this->extensions[$name]);
+        }
```

Comment:
```suggestion
        unset($this->extensions[$name]);
```

You wouldn't have to worry about the array key already existing or not.

Reply:
@derekmd, Thank you. I have changed.